common/djangoapps/course_modes/views.py

```diff
@@ -320,20 +326,34 @@ def post(self, request, course_id):
         if requested_mode == 'verified':
             amount = request.POST.get("contribution") or \
                 request.POST.get("contribution-other-amt") or 0
+            LOG.info(
+                '[Track Selection Check] Requested verified mode - contribution: [%s], contribution other amount: [%s]',
```

Comment:
_#performance_ Mental note: since this log message should be sent for every request, this line incurs ~+= 20 MB of logs. No action required.